src/dev/tslint/rules/moduleMigrationRule.js

```diff
@@ -17,21 +17,37 @@
  * under the License.
  */
 
+const path = require('path');
 const Lint = require('tslint');
 
+const KIBANA_ROOT = path.dirname(path.resolve(__dirname, '../../..'));
+
 class ModuleMigrationWalker extends Lint.RuleWalker {
   visitImportDeclaration(node) {
-
     const moduleId = node.moduleSpecifier.text;
     const mapping = this.options.find(
-      mapping => mapping.from === moduleId || mapping.from.startsWith(moduleId + '/')
```

Comment:
This was incorrect, fixed to check the module being imported instead of the lint configuration.